src/bsoncxx/private/itoa.cpp

```diff
@@ -1054,29 +1060,5 @@ void itoa::init() {
     }
 }
 
-itoa::itoa() : itoa(0) {}
-
-itoa::itoa(const itoa& rhs) : itoa(rhs._val) {}
-
-itoa& itoa::operator=(const itoa& rhs) {
-    _val = rhs._val;
-    init();
-    return *this;
-}
-
-itoa& itoa::operator=(uint32_t i) {
-    _val = i;
-    init();
-    return *this;
-}
-
-const char* itoa::c_str() const {
```

Comment:
I inlined the accessors. Silly to call a function to get these, and itoa is package private.